### ISSUE TITLE
How to copy logs from Desktop Terminal

### ISSUE THREAD

**Issue by tomasonjo** [neo4j-apps/neo4j-desktop#85]
I run the desktop terminal for a Neo4j instance to check out the logs, but I have no idea how to copy-paste them out of the Desktop terminal. Ctrl+c doesn't work, I also tried ctrl + shift + c like in ubuntu terminal and it doesn't work either.

**Comment by akollegger** [comment]
That is unexpected @tomasonjo . The text should be selectable and available for regular copy-paste operations. Is any other text copy-pasteable? Like a query from Neo4j Browser?

**Comment by tomasonjo** [comment]
Yes, copying a query from Neo4j Browser or Neo4j Bloom. I tried the CTRL + C and the CTRL + SHIFT + C, and none of them works. Maybe adding a right-click with options to copy would be cool.

I realized that I can copy from Neo4j Terminal with my middle button on the mouse, the one used for scrolling.
